pkg/storkctl/action.go

```diff
@@ -59,15 +65,54 @@ func newFailoverCommand(cmdFactory Factory, ioStreams genericclioptions.IOStream
 				}
 				_, err = storkops.Instance().CreateAction(&action)
 				if err != nil {
-					util.CheckErr(err)
-					return
+					printMsg(
+						fmt.Sprintf(
+							"failed to start failover for namespace %v due to error %v",
+							namespace, err),
+						ioStreams.ErrOut)
+					continue
+				}
+				printMsg(fmt.Sprintf("started failover for namespace %v", namespace), ioStreams.Out)
+				actions = append(actions, action)
+			}
+			for _, action := range actions {
+				isSuccessful, err := waitForActionToComplete(action.Name, action.Namespace)
```

Comment:
If we are going to block the CLI from finishing until the action completes then we need to give some kind of status back to the user. Instead I would say print a line here indicating that the action has been triggered and check the status of the action. There is no guarantee ever that the action esp if it is a failback to finish within 10 mins

```
storkctl perform failover -n ns1
Started failover for namespace ns1. To check the status of the failover run the following command:
storkctl get failoverstatus -n ns1
```